lua/cinema/entities/doom3_bfg/cl_init.lua

```diff
@@ -1,10 +1,18 @@
 ﻿-- This file is subject to copyright - contact swampservers@gmail.com for more information.
 include('shared.lua')
 
+local bfgflare = CreateMaterial( "bfgflare", "UnlitGeneric", {
```

Comment:
This doesn't have anything to do with YouTube. Are you trying to do multiple PRs at once?

Reply:
whatever I'll just change the PR title